src/qt/bitcoingui.cpp

```diff
@@ -923,6 +923,7 @@ void BitcoinGUI::message(const QString &title, const QString &message, unsigned
 
         showNormalIfMinimized();
         QMessageBox mBox(static_cast<QMessageBox::Icon>(nMBoxIcon), strTitle, message, buttons, this);
```

Comment:
Nit, since it's not possible to specify the format in the constructor, change everything to setters:
```cpp
QMessageBox mBox(this);
mBox.setIcon(static_cast<QMessageBox::Icon>(nMBoxIcon));
mBox.setWindowTitle(strTitle);
mBox.setTextFormat(Qt::PlainText);
mBox.setText(message);
mBox.setStandardButtons(buttons);
```

Reply:
Meh... let's not add more work here than is necessary. It works so it's ok, imo.